solr/solr-ref-guide/modules/deployment-guide/pages/solrcloud-distributed-requests.adoc

```diff
@@ -445,6 +445,21 @@ For example, the following line makes Solr use the `ExactStatsCache` implementat
 ----
 <statsCache class="org.apache.solr.search.stats.ExactStatsCache"/>
 ----
+When `ExactStatsCache`, `ExactSharedStatsCache` or `LRUStatsCache` implementations are enabled in `solrconfig.xml`, distributed calls to fetch global term stats apply to all queries.
+For queries that do not utilize distributed IDF for score calculation, for example
```

Comment:
to be honest, I don't understand the point of this particular sentence. Line 455 and below make sense and clear.

Reply:
Thanks Mikhail. Changed wording to make it more clear.